modules/administration/pages/tshoot-taskomatic.adoc

```diff
@@ -0,0 +1,34 @@
+[[troubleshooting-taskomatic]]
+= Troubleshooting Taskomatic
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+If taskomatic crashes or has failed to terminate a process, it can cause problems with metadata generation.
+When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
+To correct this, determine if any processes are running, or if a crash could have occurred.
```

Comment:
I have no idea about what "processes are running" means here.

Can you find the original context from this text? Bug/mailing list message/TID... I can't really propose a better text.

Maybe @mcalmer you remember?

Reply:
So, I thought it came from the wiki, and upon searching, I found this: https://wiki.microfocus.com/index.php/SUSE_Manager/CLient_Server_package_inconsistency

Not sure if it's the original, but it's close.